docs/Tutorial_DiscordMERLIN.ipynb

```diff
@@ -0,0 +1,993 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #31.        k_discord_dist = np.full(k, -1, dtype=np.float64)
</pre><p> </p><p>Can we use <code>np.NINF</code>  instead of <code>-1</code>  as the fill value? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/505/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>

Reply:
Sure. Is that just to be consistent with other parts where we used `np.Inf` / `np.NINF` for distance?

Reply:
<p>Yes!</p>